packages/react-core/src/components/HelperText/__tests__/__snapshots__/HelperText.test.tsx.snap

```diff
@@ -103,7 +103,7 @@ exports[`HelperText icon helper text renders properly 1`] = `
       >
         <svg
           aria-hidden="true"
-          class="pf-svg"
+          class="pf-svg undefined"
```

Comment:
Is the `undefined` class supposed to be there?

Reply:
I don't t see it on the actual component in docs.  I added a check to only spread the className when not `undefined`  I am not sure why it is happening with snapshots.

Reply:
Ok, I updated our version of jest.  I was seeing different results in CI vs in our workspace.  Updating jest fixed it.